sceptre/bridge/config/prod/bridge-ops-djeng.yaml

```diff
@@ -0,0 +1,16 @@
+template:
+  type: http
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.4.0/EC2/jc-ec2-linux.j2
```

Comment:
I would suggest changing this to reference `https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/EC2/basic-ec2.j2`